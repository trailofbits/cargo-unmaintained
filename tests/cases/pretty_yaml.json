[
  {
    "name": "countme",
    "outdated_deps": [
      {
        "name": "rustc-hash",
        "req": "^1.1",
        "version_latest": null,
        "version_used": "1.1.0"
      }
    ],
    "repo_status": {
      "Age": null
    },
    "version": "3.0.1"
  }
]
