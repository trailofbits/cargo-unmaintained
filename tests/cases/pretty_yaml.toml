# smoelius: `pretty_yaml` contains paths that are invalid for NTFS.
url = "https://github.com/g-plane/pretty_yaml"
rev = "v0.5.0"
