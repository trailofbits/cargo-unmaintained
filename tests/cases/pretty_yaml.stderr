Scanning 53 packages and their dependencies (pass --verbose for more information)
